packages/cli/src/utils/docker.ts

```diff
@@ -52,83 +25,52 @@ export const createNetwork = async (name: string, client = defaultClient) => {
   if (networks.length > 0) return networks[0]
   return client.network.create({
     CheckDuplicate: true,
-    Driver: 'overlay',
     Name: name,
```

Comment:
the engine's network name could be `mesg_engine` to set the same name I suggested in another comment.